.github/workflows/encrypted_settings.py

```diff
@@ -0,0 +1,64 @@
+# Settings for django_mongodb_backend/tests when encryption is supported.
+import os
+from pathlib import Path
+
+from mongodb_settings import *  # noqa: F403
+from pymongo.encryption import AutoEncryptionOpts
+
+os.environ["LD_LIBRARY_PATH"] = str(Path(os.environ["CRYPT_SHARED_LIB_PATH"]).parent)
+
+AWS_CREDS = {
+    "accessKeyId": os.environ.get("FLE_AWS_KEY", ""),
+    "secretAccessKey": os.environ.get("FLE_AWS_SECRET", ""),
```

Comment:
Comment where the environment variable names come from given that then name doesn't appear anywhere else in this repo. (Is it really configured correctly?) Maybe it's better to have some configuration variable like "USE_AWS_KMS" rather than inferring it from these values.

Reply:
> I thought it might be incorrect since the names are different from this line in evergreen config: `include_expansions_in_env: ["AWS_SECRET_ACCESS_KEY", "AWS_ACCESS_KEY_ID", "AWS_SESSION_TOKEN"]`

I believe those are for [fetching the secrets](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/README.md#usage) via `ec2.assume_role` and `FLE_*` is for KMS testing with AWS credentials.